client/my-sites/sharing/buttons/options.jsx

```diff
@@ -134,13 +134,22 @@ module.exports = React.createClass( {
 			return;
 		}
 
+		const checked = typeof this.props.values.jetpack_comment_likes_enabled === 'undefined'
```

Comment:
Yeah, I would stress readability and maintainability. This is often why I'll use `_.includes` instead of `Array#indexOf` checks even though they have similar conciseness. `_.get` can sometimes be less readable, since it may not be obvious to a new developer the behavior, especially around its third argument. In this case I think it helps express your intention better than the native approach, which is to use the property if it exists, whether falsey or not, else falling back to `false`.